### PR TITLE
New network parameter

### PR DESCRIPTION
**Pull request by mcwimm**
Add parameter `groupID` to network dictionary to use it as output variable

### CODE REVIEW

File: PopulationLib/Tree.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def iniNetwork(self):
         self.network['potential_partner'] = []
         # List with the names of trees (tree_name) with which it is connected
         self.network['partner'] = []
+        self.network['groupID'] = []
```
Comment (jbathmann) [inline PopulationLib/Tree.py:93]: this is the only change, which is not included in #118, right?

Comment (mcwimm) [reply PopulationLib/Tree.py:93]: no. see changes :)

File: TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py [hunk 2]
```diff
@@ -308,11 +309,16 @@ def makeGraphDictionary(self):
...
+                    # If the pair occurs again it wll be appended in link_list
+                    # This means that the link (or rgf process) is finished
+                    # for both trees
                     if {neighbour, vertex} not in link_list2:
                         link_list2.append({vertex, neighbour})
                     else:
-                        # trees are only put in the dict. if the occur more than
-                        # ones, i.e. both partners have finished rgf
+                        # trees are only put in the dict. if they occur more
+                        # than ones, i.e. both partners have finished rgf
```
Comment (jbathmann) [inline TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py:321]:
```suggestion
                        # than once, i.e. both partners have finished rgf
```

**Comment by jbathmann** [comment]
:+1:

File: TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py (hunk 2 above)
Comment (mcwimm) [reply TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py]: thx. fixed.

**Comment by jbathmann** [comment]
rdy to merge?!

**Pull request merged**